libgalois/include/katana/analytics/BfsSsspImplementationBase.h

```diff
@@ -336,6 +311,65 @@ struct BfsSsspImplementationBase {
 
     return true;
   }
+
+  // TODO(lhc): Now BFS and SSSP use different labels: parent and distance,
+  //            respectively. Therefore, we should differentiate worklist
+  //            wrappers. For consistency, we could put `sssp` prefix later.
```

Comment:
We may want two separate subclasses of `BfsSsspImplementationBase`. One with BFS variants of the utilities, one with SSSP. No need for prefixes.